src/Microsoft.AspNetCore.Mvc.Razor/Internal/RazorViewCompiler.cs

```diff
@@ -116,6 +116,8 @@ public class RazorViewCompiler : IViewCompiler
             }
         }
 
+        public bool WatchForChanges { get; internal set; }
```

Comment:
Is there a good reason why this isn't `get set` ? I know I would have preferred that the whole class be internal, but there's no takebacks now.